server/etc/pulp/server.conf

```diff
@@ -110,7 +113,7 @@
 [security]
 # cacert: /etc/pki/pulp/ca.crt
 # cakey: /etc/pki/pulp/ca.key
-# ssl_ca_certificate: /etc/pki/pulp/ssl_ca.crt
+# ssl_ca_certificate: /etc/pki/pulp/ssl_ca.crt #Deprecated! See above description for details.
```

Comment:
Obviously very minor, but I think that since the majority of our code is python, we should use PEP-8 compliant comments where it makes sense. So it would be my preference, though not required, that we use 2 spaces before the second `#` and one following before the comment.